frontend/app/configuration/Social.scala

```diff
@@ -1,7 +1,9 @@
 package configuration
 
 object Social {
+  val twitterUsername = "gdnmembership"
+  val twitterHashtag = "#guardianmembership"
```

Comment:
This is the one that we've been using up till now, I just moved where it's defined.